CHANGELOG.md

```diff
@@ -1,5 +1,60 @@
 # Cromwell Change Log
 
+## 28
+
+### Call Caching
+
+* New endpoint returning the hash differential for 2 calls. 
+
+`GET /api/workflows/:version/callcaching/diff`
+
+The endpoint returns a JSON response such as the following:
+
+```json
+{
+    "callA": {
+        "workflowId": "85174842-4a44-4355-a3a9-3a711ce556f1",
+        "callFqn": "wf_hello.hello",
+        "jobIndex": -1,
+        "allowResultReuse": true
+    },
+    "callB": {
+        "workflowId": "7479f8a8-efa4-46e4-af0d-802addc66e5d",
+        "callFqn": "wf_hello.hello",
+        "jobIndex": -1,
+        "allowResultReuse": true
+    },
+    "hashDifferential": [
+        {
+            "command template": {
```

Comment:
Going back to the convo on whether or not to show the original values, I'm wondering if putting the hashes is useful at all here. While we're not displaying original values, what about just saying which fields didn't match?